### DISCUSSION TITLE
Reading from Sunstonepower 10KW 48V 3-Phasen Hybrid Inverter

### DISCUSSION THREAD

**Discussion by antaril** [jblance/mpp-solar#377] [Q&A]
Hi there,

iam trying to read from my inverter via usb, but when i do : "sudo /home/pi/.local/bin/mpp-solar -p /dev/hidraw0 -c QPIGS"

i got this back: 
Command: QPIGS - General Status Parameters inquiry
--------------------------------------------------------------------------------
Parameter        Value                  Unit
ac_input_voltage 0.0                    V       {'icon': 'mdi:lightning'}

is this a problem of me, or is the inverter not compatible? Can i read from rs232?
It would be great if someone can help me!

greets

**Comment by cordelster** [comment]
You don't give allot of info so lets start with some basics.


What returns from the following commands?
`sudo /home/pi/.local/bin/mpp-solar -p /dev/hidraw0 -c QPI`
`sudo /home/pi/.local/bin/mpp-solar -p /dev/hidraw0 -c QMN`
Try maybe:
`sudo /home/pi/.local/bin/mpp-solar -p /dev/hidraw0 -c QPGS1`
